src/commons/map/geocodeAddress.ts

```diff
@@ -0,0 +1,35 @@
+const cache: Map<string, { lat: number; lng: number; expiry: number }> = new Map();
+const CACHE_TTL = 5 * 60 * 1000; // Cache entries expire after 5 minutes
```

Comment:
I'm pretty sure houses don't move, we can safely remove the invalidation mechanism, it's a demo website